cts.schema.json

```diff
@@ -45,12 +48,31 @@
           "required": [
             "document",
             "result"
-          ]
+          ],
+          "properties": {
+            "invalid_selector": false,
```

Comment:
@gregsdennis I took the liberty of tightening this up, even though it's not needed for the current PR.